main/src/ca/uwaterloo/flix/language/dbg/DocAstFormatter.scala

```diff
@@ -190,6 +190,10 @@ object DocAstFormatter {
           text("try") +: curly(bodyf) +:
             text("catch") +: curly(rs)
         )
+      case Throw(exp) =>
+        group(
+          text("throw") +: aux(exp, paren = false, inBlock = true)
```

Comment:
inBlock means that let fx doesn't get {} around it, but it should have (e.g. `throw {let x = 42; x + 1}`) so inBlock should be false.

The group(..) around it all doesn't do anything because text, +:, and, aux doesn't have "open" newlines, so that can also be removed.

Its a bit obscure

Reply:
Oh, but also this will just be deleted by my other comment, so nevermind the details